fdbserver/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ endif()
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/workloads)
 
 add_flow_target(EXECUTABLE NAME fdbserver SRCS ${FDBSERVER_SRCS})
+set_target_properties(fdbserver PROPERTIES LINK_FLAGS "-Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/fdbserver.map")
+set_target_properties(fdbserver PROPERTIES LINK_FLAGS "-Wl,--export-dynamic")
```

Comment:
Mac compilation fails because of this. From the CI logs:

```
...
agementWorkload.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/ThreadSafety.actor.g.cpp.o fdbserver/CMake
Files/fdbserver.dir/workloads/Throttling.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/Throughput.actor.
g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/ThroughputQuota.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir
/workloads/TimeKeeperCorrectness.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/TransactionCost.actor.g.c
pp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/TriggerRecovery.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/wo
rkloads/UDPWorkload.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/UnitPerf.actor.g.cpp.o fdbserver/CMake
Files/fdbserver.dir/workloads/UnitTests.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/Unreadable.actor.g
.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/ValidateStorage.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/
workloads/VersionStamp.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/WatchAndWait.actor.g.cpp.o fdbserve
r/CMakeFiles/fdbserver.dir/workloads/Watches.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/WatchesSameKe
yCorrectness.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/WorkerErrors.actor.g.cpp.o fdbserver/CMakeFil
es/fdbserver.dir/workloads/WriteBandwidth.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/WriteDuringRead.
actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/workloads/WriteTagThrottling.actor.g.cpp.o fdbserver/CMakeFiles/fdbs
erver.dir/workloads/workloads.actor.g.cpp.o fdbserver/CMakeFiles/fdbserver.dir/coroimpl/CoroFlow.actor.g.cpp.o -o bin
/fdbserver  lib/libfdbclient.a  lib/libmetacluster.a  lib/libsqlite.a  fdbserver/rocksdb-prefix/src/rocksdb-build/lib
rocksdb.a  /opt/homebrew/lib/liblz4.a  lib/libfdbclient.a  lib/libfdbrpc.a  lib/libflow.a  lib/libstacktrace.a  lib/l
ibfmt.a  lib/libcrc32.a  /opt/homebrew/opt/openssl@1.1/lib/libssl.a  /opt/homebrew/opt/openssl@1.1/lib/libcrypto.a  /
opt/boost_1_78_0/lib/libboost_context.a  /opt/boost_1_78_0/lib/libboost_filesystem.a  /opt/boost_1_78_0/lib/libboost_
atomic.a  /opt/boost_1_78_0/lib/libboost_iostreams.a  -lbz2  -lz  /opt/boost_1_78_0/lib/libboost_serialization.a  /op
t/boost_1_78_0/lib/libboost_system.a  -Xlinker -framework -Xlinker IOKit  -Xlinker -framework -Xlinker CoreFoundation
  lib/liblibb64.a  lib/libmd5.a  lib/libeio.a && :
ld: unknown option: --export-dynamic
clang: error: linker command failed with exit code 1 (use -v to see invocation)

...
```